chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,88 @@
+import sys
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from poleno.utils.misc import squeeze
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    args = u'attachment_id [file]'
+    help = squeeze(u"""
+            Creates anonymization for the specified Attachment. The content source is file, that can
+            be passed as an argument, or stdin. Preferred source is file. If file is not specified
+            and stdin is empty, the command will fail. Anonymization created this way will be marked
+            as successful. Only one successful anonymization can be assigned to the Attachment.
+            """)
+
+    option_list = BaseCommand.option_list + (
+        make_option(u'--content_type',
+                    help=squeeze(u"""
+                            Content type of file, e.g. "application/pdf". Automatically guessed from
+                            the file content if not specified.
+                            """)
+                    ),
+        make_option(u'--debug',
+                    default=u'',
+                    help=u'Debug message to the newly created anonymization. Empty by default.'
+                    ),
+        make_option(u'--force',
+                    action=u'store_true',
+                    help=squeeze(u"""
+                            The command refuses to anonymize attachment if a successful
+                            anonymization already exists. This flag disables this check. Deletes all
+                            existing successful anonymizations and creates new one. Unsuccessful
+                            anonymizations will stay unaffected.
```

Comment:
@viliambalaz By som to zjednodusil, nech sa to lepsie chape

> Overwrite any existing anonymization for the attachment.

Reply:
:ok:, `.md` dokumentaciu som tiez nechal nezmenenu.